src/openrct2/ride/RideRatings.cpp

```diff
@@ -1892,12 +1894,11 @@ static void ride_ratings_calculate_miniature_railway(Ride* ride)
     ride->upkeep_cost = ride_compute_upkeep(ride);
     ride->window_invalidate_flags |= RIDE_INVALIDATE_RIDE_INCOME;
 
-    int32_t edx = get_num_of_sheltered_eighths(ride);
-    if (((edx >> 8) & 0xFF) >= 4)
+    auto shelteredEighths = get_num_of_sheltered_eighths(ride);
+    if ((shelteredEighths.TrackShelteredEighths & 0xFF) >= 4)
```

Comment:
No need for the 0xff

Reply:
I was a bit loathe to remove it - why was it there in the first place?

Reply:
Because you had to mask off the upper 8bits. Now that the function returns a struct its not required.